packages/ssx-sdk/tests/modules/Encryption.test.ts

```diff
@@ -64,9 +76,15 @@ xdescribe('Encryption', () => {
 
     test('Should be able to encrypt and decrypt a message', async () => {
       const message = 'Hello World';
-      const encryptedMessage = await encryption.encrypt(message);
-      const decryptedMessage = await encryption.decrypt(encryptedMessage);
-      expect(decryptedMessage).toEqual(message);
+      const encryptedData = await encryption.encrypt({
+        content: message,
+        accessControlConditions: LIT_ENCRYPTION_ACCESS_CONTROL_CONDITIONS
+      });
+      const decryptedData = await encryption.decrypt({
+        ...encryptedData,
+        accessControlConditions: LIT_ENCRYPTION_ACCESS_CONTROL_CONDITIONS
+      });
+      expect(decryptedData.decryptedString).toEqual(message);
```

Comment:
I updated the tests here, but I still have trouble with the crypto library. I so left them disabled.